frontend/mr-admin-flate/src/components/skjema/ControlledDateInput.tsx

```diff
@@ -14,11 +14,7 @@ export interface DateInputProps {
   placeholder?: string;
 }
 
-export const ControlledDateInput = forwardRef(function ControlledDateInput(
-  props: DateInputProps,
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  _,
```

Comment:
Var det noe poeng med den her?

Reply:
Usikker, men det kan hende typescript har klaget på at `forwardRef` krevde at funksjonen tok inn to argumenter, uavhengig av om man brukte andre argument eller ikke :thinking: 

Vi har siden dette implementert en egen override av typedeklarasjonen til `React.forwardRef` definert i `react-overrides.d.ts` - kan hende dette påvirker denne komponenten på noe vis?